Home_Work_2_task_2

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace L_2_array_task_2
+{
+    /*Преобразовать массив так, чтобы сначала шли все отрицательные элементы, а потом
+     * положительные (0 считать положительным)*/
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            const int size = 20;
+            int[] Masivkas = new int[size];
+            Fill_Masivkas(Masivkas, size);
+            Sort_Masivkas(Masivkas, size);
+            Ready_Sort_Masivkas(Masivkas, size);
+
+            //Random rand = new Random();
+
```

Comment:
Delete useless code

Reply:
Ooo, understand all code it can be all in one main function other methods are meaningless, in this case) Right?